src/wrappers/linkify/index.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import {SFC} from 'types';
+import ReactLinkify from 'react-linkify';
+import * as S from './Styles';
+
+interface CustomLinkifyProps {
+  children: React.ReactNode;
+}
+
+const Linkify: SFC<CustomLinkifyProps> = ({children}) => {
+  const componentDecorator = (href: string, text: string, key: number) => (
```

Comment:
Added this `componentDecorator` for opening the links in a new tab.
Strangely, this package doesn't have any straight forward property to open the links in a new tab